doctrine/doctrine-migrations-bundle/3.0/config/packages/doctrine_migrations.yaml

```diff
@@ -0,0 +1,5 @@
+doctrine_migrations:
+    migrations_paths:
+        # namespace is arbitrary but should be different from App\Migrations
+        # as migrations classes should NOT be autoloaded
+        'App\DoctrineMigrations': '%kernel.project_dir%/Migrations'
```

Comment:
Maybe use `migrations` not `Migrations` as all other root dirs

Reply:
Shouldn't this be inside the src dir? Did I choose the wrong kernel parameter?

Reply:
If you want it inside `src` it should be:
`%kernel.project_dir%/src/Migrations`

Reply:
See previous [recipe](https://github.com/symfony/recipes/blob/master/doctrine/doctrine-migrations-bundle/1.2/config/packages/doctrine_migrations.yaml#L2)

Reply:
Hmm, you are right!

Now you have raised an interesting point, does it make sense to have migrations outside of the src? 

BTW, to be honest I do not remember why migrations should not be autoloadable... To not polute the class map?

Reply:
See previous discussions https://github.com/symfony/recipes/pull/87#issuecomment-305150705 (for `src `) and https://github.com/symfony/recipes/issues/718 (for `root`)
I personally vote for `root` and already change it in my projects

> to be honest I do not remember why migrations should not be autoloadable

IIRC, to prevent using it in the application

Reply:
migrations are as suggested in https://github.com/symfony/recipes/issues/718